_pages/webextensions.md

```diff
@@ -12,160 +12,173 @@ parent: activities
 <div class="col-md-3 hidden-xs pull-right" markdown="1">
 ## Impact and Metrics
 
-Add-ons provide a safe, simple, and powerful way to personalize your browsing experience. With WebExtensions APIs, it’s easier than ever to create extensions that can be easily ported to Firefox, Chrome, Edge, and Opera. Throwing a workshop helps more people learn how to develop add-ons and help ensure that Firefox remains the most customizable browser.
+Throwing a workshop helps more people learn how to develop add-ons and help ensure that Firefox remains the most customizable browser.
+
 
 ### Campaign Goals
 
 __Impact:__
 
 * 125 developers are introduced to and building browser extensions either online or at regional events.
-* 50 add-ons tagged with mozactivate17 in addons.mozilla.org.
-* 50 social media posts (Twitter, Facebook, blogs, etc.) that reference extension development and #MozActivate.
+* 20 add-ons tagged with ‘#mozactivate17’ in addons.mozilla.org.
 
 __Strength:__
 
-* 25 Mozillians organize Build Your Own Extension workshops
+* 15 Mozillians organize Build Your Own Extension workshops 
 
 ### Goals for Your Workshop
 
 * Workshop is attended by five or more people
-* 100% of attendees successfully complete Part II of the activity
-* 10% or more of attendees go on to complete Part III of the activity
+* 60% of participants from the first workshop attend at least one other event in the series
 * 100% of attendees receive the event follow-up email
 
 ## Finding Participants
 
 Your audience for this event should be __people who have written HTML and JavaScript code in the past__.
 
-People with more JavaScript experience are encouraged to help beginners get started. If you have experience developing browser extensions, you may skip the “Build Your First Extension” exercise and focus the activities listed in Part III.
+New to JavaScript and HTML? Check out these tutorials by community members [Viswaprasath Ks](https://www.youtube.com/playlist?list=PLdIyyZgtQZ7CJR5XFB5xvZH0oaBIT_gTD) and [Trishul Goel](https://www.youtube.com/playlist?list=PLI08QU9qtPEJiBs7rVOT4FvXJlisDgTlS) to build your programming skills and create browser extensions! 
+
+
 </div>
 
 <div class="col-md-9" markdown="1">
 
 {: .alert .alert-info .impactbox}
 <span class="glyphicon glyphicon-ok-circle" aria-hidden="true"></span>
-Firefox is an open-source web browser that cares about user control and privacy, and add-ons provide a safe, simple, and powerful way to personalize your browsing experience. From blocking ads to organizing tabs, add-ons help make Firefox your own.
+Firefox is an open-source web browser that cares about user control and privacy, and extensions  provide a safe, simple, and powerful way to personalize your browsing experience. From blocking ads to organizing tabs, add-ons help make Firefox your own.
```

Comment:
There is a double space between "extensions  provide". Also, is the mix between "extensions" and "add-ons" intended here?

Reply:
Yes, this is fine. :)

Reply:
the double space still needs to be fixed though ;)